pkg/minikube/registry/drvs/vfkit/vfkit.go

```diff
@@ -104,7 +109,7 @@ func configure(cfg config.ClusterConfig, n config.Node) (interface{}, error) {
 func status() registry.State {
 	_, err := exec.LookPath("vfkit")
 	if err != nil {
-		return registry.State{Error: err, Fix: "Run 'brew tap cfergeau/crc && brew install vfkit'", Doc: docURL}
+		return registry.State{Error: err, Fix: "Run 'brew install vfkit'", Doc: docURL}
```

Comment:
:+1: 
vfkit in cfergeau/crc will no longer be updated now that it’s in the main repo. I should remove it from this personal tap.

Reply:
sure

Reply:
From https://github.com/crc-org/vfkit/commit/1228143e04eee309db63d99da03cedcf54cc3ed3